### PR TITLE
change readme file

### PR DESCRIPTION
**Pull request by bicotrucarduino**
change readme file comment

### CODE REVIEW

**Comment by bicotrucarduino** [comment]
sdfsdf

File: README.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+bicotrucarduino change to pull request
+
```
Comment (VoLinhTruc) [inline README.txt:2]: remove this line

Comment (bicotrucarduino) [reply README.txt:2]:
```suggestion
int a;

```

Comment (bicotrucarduino) [reply README.txt:2]: empty is no problem

Comment (VoLinhTruc) [reply README.txt:2]: pls remove

File: README.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
+bicotrucarduino change to pull request
```
Comment (bicotrucarduino) [inline README.txt:1]: this line may ok

File: README.txt (hunk 1 above)
Comment (VoLinhTruc) [inline README.txt:2]: this line may fine

File: README.txt (hunk 2 above)
Comment (VoLinhTruc) [inline README.txt:1]: this line may fine

Comment (VoLinhTruc) [inline README.txt:1]: remove the bicotrucarduino

File: README.txt (hunk 1 above)
Comment (VoLinhTruc) [inline README.txt:2]: remove this line

**Review by VoLinhTruc — COMMENTED** [review]
pls chekc this review

File: README.txt (hunk 1 above)
Comment (VoLinhTruc) [inline README.txt:2]: remove this line

Comment (VoLinhTruc) [inline README.txt]: remove line 2

**Review by VoLinhTruc — CHANGES_REQUESTED** [review]
remove line 2

File: README.txt (hunk 1 above)
Comment (VoLinhTruc) [inline README.txt]: add "end file" at last line

File: README.txt (hunk 2 above)
Comment (VoLinhTruc) [inline README.txt:2]: add "begin" at first line

**Review by VoLinhTruc — CHANGES_REQUESTED** [review]

**Pull request closed without merging**